src/System.CommandLine.Subsystems/Subsystems/PipelineResult.cs

```diff
@@ -13,4 +15,14 @@ public class PipelineResult(ParseResult? parseResult, string rawInput, Pipeline?
     public bool AlreadyHandled { get; set; }
     public int ExitCode { get; set; }
 
+    public void NotRun(ParseResult? parseResult)
+    {
+        // no op because defaults are false and 0
+    }
```

Comment:
If this is going to be a no-op should we simply remove the method as not calling it does the same thing?

Reply:
Created an issue on this: #2443